src/site/layouts/vet_center.drupal.liquid

```diff
@@ -0,0 +1,51 @@
+{% include "src/site/includes/header.html" with drupalTags = true %}
+{% include "src/site/includes/alerts.drupal.liquid" %}
+{% include "src/site/includes/preview-edit.drupal.liquid" %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" with hideHomeBreadcrumb = false %}
+
+<div class="interior" id="content">
+  <main class="va-l-detail-page va-facility-page">
+    <div class="usa-grid usa-grid-full">
+      <div class="usa-width-three-fourths">
+        <article class="usa-content va-l-facility-detail">
+          {% if entityLabel != empty %}
+            <h1>{{ entityLabel }}</h1>
```

Comment:
i'm not familiar with this entity (obviously) but typically there is a `entity.title` property we use for the first one. There is some logic in the header file that might be worth checking out to make sure this works okay with the meta tags. Or just check the page source of this page.`src/site/includes/metatags.drupal.liquid`

Reply:
also i feel like the `h1` should be required here?

Reply:
Thanks @ncksllvn - we'll check with the CMS folks on this. This is just a quick "spike" to get the page started; we have a lot more work to do on this!